src/EditorFeatures/CSharpTest/EditAndContinue/BreakpointSpansTests.cs

```diff
@@ -247,6 +243,245 @@ IEnumerable<int> Goo()
 }");
         }
 
+        [Fact]
+        public void GetBreakpointSequence7()
+        {
+            TestAll(@"
+class C
+{
+    IEnumerable<int> Goo()
+    $$[|{|]
+        [|_ = M2(
+            c
+            switch
```

Comment:
I notice that there is no annotation around the `switch { ... }` portion. Is that because this test only shows a certain kind of sequence points?

Reply:
That is not a place where you can set a breakpoint.  Its only purpose is to show where the program is if you stop on a method called by the state machine and look up the stack.

---
In reply to: [398875995](https://github.com/dotnet/roslyn/pull/42711#discussion_r398875995) [](ancestors = 398875995)